src/main/java/org/folio/orders/utils/ErrorCodes.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.orders.utils;
+
+public enum ErrorCodes {
+
+  PO_NUMBER_ALREADY_EXISTS("poNumberNotUnique", "PO Number already exists"),
+  UNDEFINED_ERROR("9999", "Undefined error");
```

Comment:
Why use an arbitrary numeric code here?

Reply:
It's just a draft code not used anywhere. Removed it from here until we reach agreement about list of codes with ui guys